### PR TITLE
Update Thanksgiving Day app to dynamically calculate days till thanksgiving

### PR DESCRIPTION
**Pull request by savdagod**
Current logic is hard coded for 2024 and 2025. This update changes the logic to calculate the days indefinitely.

### CODE REVIEW

**Comment by savdagod** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/thanksgivingday/thanksgiving_day.star [hunk 1]
```diff
@@ -16,29 +16,30 @@ R0lGODlhIAAgAPcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
...
 
-    if now.month > 11:
-        thanksgiving_year = now.year + 1
-
-    thanksgiving = time.time(year = thanksgiving_year, month = 11, day = thanksgiving_day, hour = 0, minute = 0, location = timezone)
-
-    print(thanksgiving)
-    days_til_thanksgiving = math.ceil(time.parse_duration(thanksgiving - now).seconds / 86400)
-    print(days_til_thanksgiving)
+    timezone = config.get("timezone") or "America/New_York"
```
Comment (IngmarStein) [inline apps/thanksgivingday/thanksgiving_day.star:30]: Should this be `timezone = config.get("$tz", "America/New_York")`?

Comment (savdagod) [reply apps/thanksgivingday/thanksgiving_day.star:30]: I’m gonna be honest, I have no idea. Looks to be that way ($tx) in some of the other countdown apps (ie Christmas) but it’s this way (timezone) in the Halloween countdown. Not sure which way is the correct way.

Comment (IngmarStein) [reply apps/thanksgivingday/thanksgiving_day.star:30]: The Halloween countdown app was fixed in https://github.com/tidbyt/community/pull/2854.

Comment (savdagod) [reply apps/thanksgivingday/thanksgiving_day.star]: Ah i see, I'll go ahead and update it then

**Pull request merged**